planetiler-core/src/main/java/com/onthegomap/planetiler/mbtiles/Mbtiles.java

```diff
@@ -234,14 +232,39 @@ public Mbtiles createTables() {
         TILES_DATA_TABLE, TILES_SHALLOW_TABLE, TILES_SHALLOW_COL_DATA_ID, TILES_DATA_TABLE, TILES_DATA_COL_DATA_ID
       ));
     } else {
-      ddlStatements.add("create table " + TILES_TABLE + " (" + TILES_COL_Z + " integer, " + TILES_COL_X + " integer, " +
-        TILES_COL_Y + ", " + TILES_COL_DATA + " blob);");
+      // here "unique" is much more compact than a "primary key without rowid" because the tile data is part of the table
+      String tilesUniqueAddition = skipIndexCreation ? "" : """
+        , unique(%s,%s,%s)
```

Comment:
is there a difference between `unique` and `primary key` here?

Reply:
afaik no - other than the usual stuff that you can have only one PK and cannot insert NULL
=> let me change this to primary key since it expresses its purpose better

Reply:
This PR looks ready to merge - did you want to change this from `unique` to `primary key` though? I'm fine either way.